aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -46,7 +46,7 @@ module aptos_framework::account {
     }
 
     struct CapabilityOffer<phantom T> has store { for: Option<address> }
-    struct RotationCapability has drop, store { account: address }
+    struct RotationCapability has key, drop, store { account: address }
```

Comment:
I didn't want to make this key. I intentionally want folks to think about where they want to place this and not make everything a resource. We can go into details tomorrow.